docs/generate.jl

```diff
@@ -17,7 +17,7 @@ for model in MODELS
             script = Literate.script(input, GENERATEDDIR)
             code = strip(read(script, String))
             mdpost(str) = replace(str, "@__CODE__" => code)
-            Literate.markdown(input, GENERATEDDIR, postprocess = mdpost)
+            Literate.markdown(input, GENERATEDDIR, postprocess = mdpost, credit = false)
             Literate.notebook(input, GENERATEDDIR, execute = true)
```

Comment:
does the option also apply for `Literate.notebook`?

Reply:
Yes, it does!